### DISCUSSION TITLE
"0-30/10 20-22 * * * " Cron Exp Error

### DISCUSSION THREAD

**Discussion by john2m** [quartznet/quartznet#989] [Q&A] [answered]
"0-30/10 20-22 * * * "  gives me Format Exception.

**Comment by lahma** [comment, marked as answer]
Instead, could you tell what you are trying to achieve?

**Reply by john2m** [reply]
So this one worked "0-30/10 20-22 * * ? "  , I m New to cron expression.  Trying to schedule the job every 10 min from 1 to 30th minute from 8 pm to 11 pm

**Reply by lahma** [reply]
Great that you got it sorted, there's also cronmaker.com to help.
